crates/cubecl-core/src/frontend/operation/base.rs

```diff
@@ -14,8 +14,8 @@ pub(crate) fn binary_expand<F>(
 where
     F: Fn(BinaryOperator) -> Operator,
 {
-    let lhs_var: Variable = *lhs;
-    let rhs_var: Variable = *rhs;
+    let lhs = lhs.consume();
```

Comment:
Ok so I guess consume reduces the ref count potentially making the variable reusable, removing the need to manually check `can_mut`